app.js

```diff
@@ -0,0 +1,11 @@
+const express = require("express");
+const { getTopics } = require("./controllers/topicsController");
+const { handle404 } = require("./errors");
+
+const app = express();
+
+app.use(express.json());
```

Comment:
We don't need this yet as we aren't posting any data. Be careful not to add code outside the scope of the tickets.